frontend/app/CampaignActionDetail.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import Loader from 'react-loader'
+import moment from 'moment'
+import API from './API'
+import CampaignCallPreview from './CampaignCallPreview'
+
+const DATE_FORMAT = 'h:mma on M/DD/YYYY'
+
+export default class CampaignActionDetail extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      action: {},
+      loaded: false
+    }
+    this.fetchCampaignAction = this.fetchCampaignAction.bind(this)
+  }
+
+  componentWillMount() {
+    this.fetchCampaignAction()
+  }
+
+  fetchCampaignAction() {
+    return API.campaignAction(this.props.params.actionId).then(data => this.setState({ action: data, loaded: true }))
+  }
+
+  get breadcrumbTitle() {
+    return this.state.action.title || this.props.params.actionId
+  }
+
+  get representativesCount() {
+    switch (this.state.action.type) {
+      case 'CampaignCall':
+        return this.state.action.matchingRepresentatives.length
+      case 'CampaignUpdate':
+        return this.state.action.campaignCall.matchingRepresentatives.length
+      default:
+        return 0
+    }
+  }
+
+  get statistics() {
+    const userConversationsCount  = this.state.action.userConversations ? this.state.action.userConversations.length : 0
+    const stats = [
+      { label: 'Members targeted', value: this.representativesCount },
+      { label: 'Constituents sent', value: userConversationsCount }
+    ]
+
+    if (this.state.action.type === 'CampaignCall') {
```

Comment:
in the future, I would move this to the backend.

maybe have a getCampaignActionDetail endpoint which returns both the campaignAction and stats and needed stats about the campaignAction